packages/react-button/etc/react-button.api.md

```diff
@@ -318,7 +318,7 @@ export { useMenuContext }
 export const useSplitButton: (props: SplitButtonProps, ref: React.Ref<HTMLElement>, defaultProps?: SplitButtonProps | undefined) => SplitButtonState;
 
 // @public (undocumented)
-export const useSplitButtonClasses: (state: SplitButtonState, options?: import("@fluentui/react-theme-provider").UseStylesOptions | undefined) => void;
+export const useSplitButtonClasses: (state: SplitButtonState, options?: import("@fluentui/react-theme-provider/lib/makeStyles.compat").UseStylesOptions | undefined) => void;
```

Comment:
Is react-button used anywhere yet? I don't like how we're collecting a lot of baggage before we have a complete set of clean converged components. Example, we have a `/next` dir in compose and multiple implementations of converged ideas, including still having SCSS files in the repo in converged components.

I'd much rather prefer that we have a single set of clear components and utilities that we ship as converged.

Reply:
> Is react-button used anywhere yet?

As I know - yes, and even if it's not used I would like to avoid just breaking things. We will remove this code once we will migrate `react-button` and other components (`react-text`, `react-cards`) to a new version of `makeStyles()`.